### DISCUSSION TITLE
Missing X64 build for Windows

### DISCUSSION THREAD

**Discussion by vhick** [c0re100/qBittorrent-Enhanced-Edition#338] [General]
Hi,

May I ask if "qbittorrent_enhanced_4.4.1.10_x64_setup.exe" is still available or it is substitute by "qbittorrent_enhanced_4.4.1.10_Qt6_x64_setup.exe"?

I didn't see the file in release section.

Thank you.

**Comment by c0re100** [comment]
If you're not HiDPI user, then use Qt5 version
https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.4.1.10/qbittorrent_enhanced_4.4.1.10_Qt5_x64_setup.exe

**Reply by vhick** [reply]
Now resolve the flickering of my screen when qBittorrent is in the foreground. Thank you very much.
